.github/workflows/inductor.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           { config: "inductor_timm_dynamic", shard: 2, num_shards: 2, runner: "linux.g5.4xlarge.nvidia.gpu" },
           { config: "inductor_torchbench_dynamic", shard: 1, num_shards: 1, runner: "linux.g5.4xlarge.nvidia.gpu" },
           { config: "inductor_distributed", shard: 1, num_shards: 1, runner: "linux.g5.12xlarge.nvidia.gpu" },
+          { config: "inductor_torchbench_multiprocess", shard: 2, num_shards: 2, runner: "linux.g5.4xlarge.nvidia.gpu" },
```

Comment:
If you want a multi-gpu runner, you need to use `linux.g5.12xlarge.nvidia.gpu`, or simply merge with the `inductor_distributed` workflow (which seems only run a very small set of jobs now).